lib/phoenix_starter_web/live/user_settings/index.html.heex

```diff
@@ -1,26 +1,25 @@
 <h1>Settings</h1>
 
 <nav>
-  <%= live_patch "Update profile", to: Routes.user_settings_path(@socket, :profile) %>
-  <%= live_patch "Update email", to: Routes.user_settings_path(@socket, :email) %>
-  <%= live_patch "Update password", to: Routes.user_settings_path(@socket, :password) %>
+  <%= live_patch("Update profile", to: Routes.user_settings_path(@socket, :profile)) %>
```

Comment:
I guess the `Phoenix.LiveView.HTMLFormatter` added these parens all over the place? I don't like it but I suppose we should abide by the formatter.